docs/reference/contributing/connectivity/CellularInterface.md

```diff
@@ -87,9 +58,9 @@ typedef enum {
 
 You must define all pins. Implement `onboard_modem_api.h`. The target board must provide an implementation of the [onboard_modem_API](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/onboard__modem__api_8h_source.html).
 
-#### Modifying cellular targets
+### Modifying cellular targets
 
-Setup for some predefined targets is available in [CellularTargets.h](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_cellular_targets_8h_source.html). For a new target, you need to modify `CellularTargets.h`. In `CellularTargets.h`, you need to specify a cellular device in addition to UART pins connecting the Mbed OS CPU to the modem. For example, if the new device MY_NEW_TARGET defined above in the "Adding modem target support" chapter is connected with QUECTEL BG96, you would need the following changes marked in between `// !!!!` in `CellularTargets.h`:
+Setup for some predefined targets is available in [CellularTargets.h](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_cellular_targets_8h_source.html). For a new target, you need to modify `CellularTargets.h` by specifying a cellular device in addition to UART pins connecting the Mbed OS CPU to the modem. For example, if the new device `MY_NEW_TARGET` defined in the "Adding modem target support" section is connected with QUECTEL BG96, you would need the following changes marked in between `// !!!!` in `CellularTargets.h`:
```

Comment:
I know some of this is existing content, but we try not to include board-specific information and examples in our documentation. Is there any way to remove some of the board-specific content and replace it with more generic instructions?

Reply:
True, this porting page could be rewritten in more neutral tone of voice. I will look into this in the next phase after Handbook has been restructured.